com.unity.multiplayer.mlapi/Editor/MLAPIProfilerModule.cs

```diff
@@ -52,13 +52,13 @@ private class MLAPIModules
 
         private static List<MLAPIProfilerCounter> CreateRPCCounters() => new List<MLAPIProfilerCounter>()
```

Comment:
perhaps?
```suggestion
        private static List<MLAPIProfilerCounter> CreateMessageCounters() => new List<MLAPIProfilerCounter>()
```

Reply:
This might be specific to just RPCs?

Reply:
Based on the results of other comments, these counters are going back to being RPC-specific, so I'm keeping the name as-is.